samples/scripts/build-logs

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+if [ $# -eq 1 ]; then
+    BUILD_RUN_NAME=$1
+    FOLLOW=0
+elif [ $# -eq 2 ] && [ "$2" = "--follow" ]; then
+    BUILD_RUN_NAME=$1
+    FOLLOW=1
+else
+    echo "Usage: build-logs <BUILD_RUN_NAME> [--follow]"
+    exit 1
+fi
+
+BUILD_RUN_NAME=$1
+
+# Retrieve the buildrun
+
+BUILD_RUN=`kubectl get buildrun ${BUILD_RUN_NAME} -o json 2>/dev/null`
+
+if [ $? -ne 0 ]; then
+    echo "A build run with the name '${BUILD_RUN_NAME}' cannot be found."
+    exit 2
+fi
+
+echo "Build run: ${BUILD_RUN_NAME}"
+
+# Extract the task run name from the buildrun
+
+set -e
```

Comment:
no need of this one here, it should always go at the top level, something like `set -euo pipefail`(recommended standard), see https://sipb.mit.edu/doc/safe-shell/

Reply:
Done.